packages/blade/src/components/Tooltip/_decisions/decisions.md

```diff
@@ -0,0 +1,199 @@
+# Tooltip API Decisions <!-- omit in toc -->
+
+The tooltip typically provides additional context about the element or its function. A tooltip is always triggered by a `mouse hover` on desktop and `on tap` on mobile.
+
+<img src="./tooltip-thumbnail.png" width="380" alt="" />
+
+- [Design](#design)
+- [`Tooltip` API](#tooltip-api)
+  - [Examples:](#examples)
+    - [Usage](#usage)
+  - [Working with non-interactive triggers](#working-with-non-interactive-triggers)
+  - [Custom Triggers](#custom-triggers)
+- [Library](#library)
+  - [Bundle Size](#bundle-size)
+- [Implementation detail nuances:](#implementation-detail-nuances)
+- [Motion](#motion)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=40540-559304&t=tmTrf3xJU6oj59fM-0) to all variants of the Tooltip component
+
+## `Tooltip` API
+
+| Prop               | Type                                                                     | Default     | Description                                                                                                                                              | Required |
+| ------------------ | ------------------------------------------------------------------------ | ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| content            | `string`                                                                 | `undefined` | Content of the tooltip                                                                                                                                   | ✅       |
+| children           | `React.ReactNode`                                                        | `undefined` | Trigger component for tooltip, Accepts any interactive element or icons                                                                                  | ✅       |
+| placement          | `top, top-start, top-end, left, right, bottom, bottom-start, bottom-end` | `top`       | Placement of tooltip                                                                                                                                     |          |
```

Comment:
also, will this position change if there's no space on the edge of the screen? for eg: we mention placement as right but there's no space. what happens then? lets mention that in the description

Reply:
Yes it will flip to the other side based on the available space. 

Not in favour of `position` sounds like our css styled prop. placement is more accurate imo.

Reply:
+1 to not calling it `position`. Since we do actually support `position` in styled-props and it has values of `relative`, `absolute`